File: MAS - project/client/src/app/_models/dishDietPlan.ts
import { DietPlan } from "./dietPlan";
import { Dish } from "./dish";

export interface DishDietPlan {
  idDishDietPlan: number;
  idDietPlan: number;
  idDish: number;
  dish: Dish;
  dietPlan: DietPlan;
}

File: MAS - project/API/API/Entities/Plans/Diet/DishDietPlan.cs
namespace API.Entities.Plans.Diet
{
    public class DishDietPlan
    {
        public virtual int IdDishDietPlan { get; set; }
        public virtual int IdDietPlan { get; set; }
        public virtual int IdDish { get; set; }
        public virtual Dish Dish { get; set; } = null!;
        public virtual DietPlan DietPlan { get; set; } = null!;
    }
}
